structuredefinitions/UKCore-DiagnosticReport.xml

```diff
@@ -2,7 +2,7 @@
 <StructureDefinition xmlns="http://hl7.org/fhir">
   <id value="UKCore-DiagnosticReport" />
   <url value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-DiagnosticReport" />
-  <version value="2.2.0" />
+  <version value="2.3.0" />
```

Comment:
we probably want to uplift these dates as well to make it easier for us to spot pre sprint 6 changes (2023-02-14) and post changes (2023-04-28)